### ISSUE TITLE
Save and load masks

### ISSUE THREAD

**Issue by dylanmcreynolds** [mlexchange/mlex_highres_segmentation#160]
We have work to do to save and load masks, possibly in Tiled.

Do we want to save both the json-ified SVC and the rasterized array every time?

**Comment by Wiebke** [comment]
The export will need to convert both json-ified SVG and rasterize array such that it contains consecutive class labels and -1 for unlabelled data.

**Comment by Wiebke** [comment]
Somes notes regarding the exporting based on things that we learned along the way: 
Different machine learning may treat unlabeled data in different ways. 

Assume we have $n$ classes,  the random forest model in scikit-learn, e.g. when used in conjunction with [skimage](https://scikit-image.org/docs/stable/api/skimage.future.html#skimage.future.fit_segmenter) expects unlabeled data to have label $0$ and classes have labels $\geq1$ , neural networks in dlsia and generally those making use of [`orch.nn.CrossEntropyLoss`](https://pytorch.org/docs/stable/generated/torch.nn.CrossEntropyLoss.html) appear to assume class labels to be equivalent to indices and labels should thus be within $[0, \dots n-1]$. The label for, -1 is a sensible value. 

To support both cases, I plan to record the "class label" for unlabeled data and number of actual classes as meta-data, so model code can make an informed decision whether it needs to transform the data.
